.github/workflows/rucio-release-images.yml

```diff
@@ -0,0 +1,41 @@
+name: Rucio Release Image CI
+
+on:
+  push:
+    tags:
+      - "release-*.cms*"
+
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [rucio-probes, rucio-daemons, rucio-server]
```

Comment:
Need rucio-ui here too

Reply:
Hello, I excluded the WebUI here, because of this: https://github.com/rucio/containers/issues/229
The UI patch releases will be independent of the rucio release. Given the new UI will be used for the first time, we might need to upgrade them almost along with rucio. 
I do not know how we deal with patch releases, so if you think, we can/should have the UI here, I will add it.

Reply:
Hi @ericvaandering

Reply:
Added rucio-ui image too.